modules/apps/foundation/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/portlet/action/ManageUserAssociatedDataMVCActionCommand.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.users.admin.web.portlet.action;
+
+import com.liferay.portal.kernel.portlet.bridges.mvc.BaseMVCActionCommand;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCActionCommand;
+import com.liferay.portal.kernel.servlet.SessionErrors;
+import com.liferay.portal.kernel.util.Constants;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.user.associated.data.service.registry.UADRegistry;
+import com.liferay.users.admin.constants.UsersAdminPortletKeys;
+
+import javax.portlet.ActionRequest;
+import javax.portlet.ActionResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Raymond Augé
+ * @author Roberto Díaz
+ */
+@Component(
+	immediate = true,
+	property = {
+		"javax.portlet.name=" + UsersAdminPortletKeys.MY_ORGANIZATIONS,
+		"javax.portlet.name=" + UsersAdminPortletKeys.USERS_ADMIN,
+		"mvc.command.name=/users_admin/manage_user_associated_data"
+	},
+	service = MVCActionCommand.class
+)
+public class ManageUserAssociatedDataMVCActionCommand
+	extends BaseMVCActionCommand {
+
+	protected void autoAnonymizeAll(ActionRequest actionRequest)
+		throws Exception {
+
+		long userId = ParamUtil.getLong(actionRequest, "selUserId");
+
+		if (userId > 0) {
+			_uadRegistry.notify(userId);
+		}
+	}
+
+	@Override
+	protected void doProcessAction(
+			ActionRequest actionRequest, ActionResponse actionResponse)
+		throws Exception {
+
+		String cmd = ParamUtil.getString(actionRequest, Constants.CMD);
+
+		try {
+			if (cmd.equals("anonymize")) {
```

Comment:
Wouldn't it be better to have an AnonymizeUserAssociatedDataMVCActionCommand and avoid using "cmd"?